defaults/main.yml

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: MIT
+---
+# The provider for nbde_client role. Currently 'clevis' is supported.
+nbde_client_provider: clevis
+
+# nbde_client_bindings is a list, and its elements support the following keys:
+#
+# * device             - specifies the path of an encrypted device in the
+#                        managed host. This device must be already configured
+#                        as a LUKS device before using the role (REQUIRED).
+# * pass               - specifies a valid passphrase for opening/unlocking
+#                        the specified device.
+# * keyfile            - specifies either the absolute or relative path, in
+#                        the control node, of a keyfile valid for
+#                        opening/unlocking the specified device.
+#  * state             - specifies whether a binding with the configuration
+#                        described should be added or removed. Setting state
+#                        to present (the default) means a binding will be
+#                        added; setting state to absent means a binding will
+#                        be removed from the device/slot.
+#
+# * slot               - specifies the slot to use for the binding. Default
+#                        is 1.
+# * servers            - specifies a list of servers to bind to. To enable
+#                        high availability, specify more than one server here.
+# * threshold          - specifies the threshold for the Shamir Secret Sharing
+#                        (SSS) scheme that is put in place when using more
+#                        than one server. When using multiple servers,
+#                        threshold indicates how many of those servers should
+#                        succeed, in terms of decryption, in order to complete
+#                        the process of recovering the LUKS passphrase to open
+#                        the device.
+# * passphrase_temporary - If yes, the passphrase that was provided via the
+#                          pass or keyfile arguments will be used to unlock
+#                          the LUKS device and then it will be removed from
+#                          the LUKS device after the binding operation
+#                          completes, i.e. it will not be valid anymore. To
+#                          be used if device has been previously created with
+#                          a dummy passphrase (for example by an automated
+#                          install like kickstart that set up some sort of
+#                          "default" password), which the role should
+#                          replace by a stronger one.
+
+# Example:
+# nbde_client_bindings:
+#    - device: /dev/sda1
+#      keyfile: /vault/keyfile
+#      state: present
+#      slot: 2
+#      threshold: 1
+#      passphrase_temporary: no
+#      servers:
+#        - http://server1.example.com
+#        - http://server2.example.com
```

Comment:
Is it necessary to reproduce the whole reference documentation here? I think it would be enough to describe the variable in a single sentence. @richm what do you think?

Reply:
That's what I was going to ask. I think a single sentence would suffice here, and the README is the actual documentation.

Reply:
Almost none of the other system roles have extensive examples in `defaults/main.yml` (I'm guilty of this in kernel_settings to some extent :-| )